packages/api/scripts/testing_logs/e2c45bc9-b0cd-40b8-b111-ba8c0fecdd3c

```diff
@@ -0,0 +1,3852 @@
+Started by upstream project "node-test-binary-windows-js-suites" build number 4164
+originally caused by:
+ Started by upstream project "node-test-commit-windows-fanned" build number 36486
+ originally caused by:
+  Started by upstream project "node-test-commit" build number 38763
+  originally caused by:
+   Started by upstream project "node-test-pull-request" build number 31671
+   originally caused by:
+    Started by user Robert Nagy
+Running as SYSTEM
+[EnvInject] - Loading node environment variables.
+Building remotely on test-rackspace-win2012r2_vs2017-x64-3 (win2012r2-vs2017 win2012r2-COMPILED_BY-vs2019-x86 win-vs2017 win2012r2-COMPILED_BY-vs2015 win-vs2017-x86 win2012r2-COMPILED_BY-vs2017-x86 win2012r2 win2012r2-COMPILED_BY-vs2015-x86) in workspace C:\workspace\node-test-binary-windows-js-suites
+Run condition [Always] enabling prebuild for step [[Execute Windows batch command, Execute shell]]
+[node-test-binary-windows-js-suites] $ cmd /c call C:\Users\ADMINI~1\AppData\Local\Temp\jenkins2320858532710216097.bat
+
+C:\workspace\node-test-binary-windows-js-suites>set SCRIPTS_REPO=nodejs/build 
+
+C:\workspace\node-test-binary-windows-js-suites>set SCRIPTS_BRANCH=master 
+
+C:\workspace\node-test-binary-windows-js-suites>curl -LO https://raw.githubusercontent.com/nodejs/build/master/jenkins/scripts/windows/ci-run.cmd 
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  0   731    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0100   731  100   731    0     0    822      0 --:--:-- --:--:-- --:--:--   821
+
+C:\workspace\node-test-binary-windows-js-suites>call ci-run.cmd node-test-binary-windows-js-suites 
```

Comment:
I'd rather not check in these files, I think you just need to update your `.gitignore` to be something along the lines of:

```
**/testing_logs
```

You can cleanup by going `git rm --force  packages/api/scripts/testing_log`